lib/SPIRV/libSPIRV/SPIRVError.cpp

```diff
@@ -0,0 +1,56 @@
+//===- SPIRVError.cpp - SPIR-V error code and checking ----------*- C++ -*-===//
+//
+//                     The LLVM/SPIRV Translator
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+// Copyright (c) 2014 Advanced Micro Devices, Inc. All rights reserved.
```

Comment:
```suggestion
// Copyright (c) 2024 Intel Corporation. All rights reserved.
```

Reply:
Neither of them! Please use Khronos copy rights from https://github.com/KhronosGroup/SPIRV-LLVM-Translator/issues/80